src/pages/blog/side-cta/_style.js

```diff
@@ -0,0 +1,71 @@
+import styled from 'styled-components'
+import Shape from './images/shape.svg'
+import { Flex } from 'components/containers'
+import device from 'themes/device'
+import { LinkButton } from 'components/form'
+
+export const SideContainer = styled(Flex)`
+    padding: 20px;
+    justify-content:start;
+
+    @media ${device.tablet} {
+        justify-content:center;
+    }
+`
```

Comment:
We can pass all this in as props. For the media query, we can pass in an object with styles to `tablet={{ justifyContent: 'center' }}`